apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,10 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
```

Comment:
use this as a reference for the values you can set: https://github.com/helm/charts/blob/master/stable/cluster-autoscaler/values.yaml

Reply:
the values you might need to add:

```yaml
autoDiscovery:
  clusterName: {{ .Values.jxRequirements.cluster.clusterName }}
awsRegion: {{ .Values.jxRequirements.cluster.region }}
rbac:
  serviceAccountAnnotations:
    eks.amazonaws.com/role-arn: arn:aws:iam::{{ .Values.jxRequirements.cluster.project }}:role/{{ .Values.jxRequirements.cluster.clusterName }}-cluster-autoscaler-cluster-autoscaler
```